contracts/actions/compoundV3/helpers/CompV3Helper.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity =0.8.10;
+
+import '../../../interfaces/compoundV3/IComet.sol';
```

Comment:
either keep IComet here and remove it from actions since they extend Helper or remove from here since it isn't used here
I think ICometExt can be removed from everywhere